wiki/Do_you_really_want_to_ask_peppy/ro.md

```diff
@@ -0,0 +1,15 @@
+# Chiar dorești să-l întrebi pe peppy?
+
+Salutare! peppy înțelege dorința ta de a dori să-l contactezi personal, dar te roagă să înțelegi mai întâi că acesta **nu** este locul unde vrei să-i ceri ajutor sau sprijin. El alege să păstrează asistența osu! separat de mesajele personale și de asemenea, consideră că este destul de greu să răspundă la PM-urile de pe forum. Prin urmare, vă solicită să faceți una dintre următoarele opțiuni:
```

Comment:
"Salutare! peppy îți înțelege dorința de a-l contacta personal, dar mai întâi te roagă să înțelegi faptul că acesta **nu** este locul pentru a-i cere ajutor sau sprijin. El preferă să păstreze asistența pentru osu! separată de mesajele lui personale și de asemenea, consideră că este destul de greu să țină pasul și să răspunda mesajelor private de pe forum. Prin urmare, vă solicită să alegeți una dintre următoarele opțiuni:"

Reply:
aprob schimbarile lui @Teetoos98